incuna-sass/functions/_rem.sass

```diff
@@ -5,6 +5,8 @@ $use-rem: false !default
     @if $use-rem
         @if $target == 0
             @return 0
+        @if $target == 1px
+            @return 1px
```

Comment:
`2px` would only be a problem with a `$font-base` much below `8px`, which is unlikely to happen.